src/components/form/select/_select.scss

```
@@ -38,4 +38,10 @@
     color: $euiTextColor;
     background: transparent;
   }
+
+  &:-moz-focusring {
+        color: transparent;
+        text-shadow: 0 0 0 $euiColorFullShade;
```

It sounds like this is sort of a known hack for fixing this issue, which is ok.

Though, the "rendered" text color should then match that of our text via `$euiTextColor` not full black.
```suggestion
        text-shadow: 0 0 0 $euiTextColor;
```

---

Please also fix your spacing. There should only be on tab stop (2 spaces) per indentation. Just follow the pattern of the rest of the file.

---

You can commit all these changes to this same PR without opening a new one.